src/js/api/order.js

```diff
@@ -2456,6 +2458,8 @@ ripe.Ripe.prototype._importOrder = function(ffOrderId, options = {}) {
     if (Object.keys(parts).length > 0) contents.parts = parts;
     if (size) contents.size = size;
     if (factory) contents.factory = factory;
+    if (domain) contents.domain = domain;

```

Comment:
try to sort the const declarations and ifs to be the same, meaning
if we're declaring `domain` and `sku` before `model`, lets add their if clauses before the `model`'s if clause

Reply:
Likewise, we are declaring it after factory. The order is already wrong. I was trying not to change to many lines in this small PR

Reply:
Can you fix the ordering on a separate PR please?

Reply:
Thanks